sdk/cosmos/mgmt/src/main/java/com/azure/management/cosmosdb/implementation/CosmosDBAccountImpl.java

```diff
@@ -388,8 +388,7 @@ public CosmosDBAccountImpl withoutReadReplication(Region region) {
         this.ensureFailoverIsInitialized();
         for (int i = 1; i < this.failoverPolicies.size(); i++) {
             if (this.failoverPolicies.get(i).locationName() != null) {
-                String locName = this.failoverPolicies.get(i).locationName().replace(" ", "")
-                    .toLowerCase(Locale.ROOT);
+                String locName = this.formatLocationName(this.failoverPolicies.get(i).locationName());
```

Comment:
Any detail?

Reply:
I just refactor it to a function since it would use every where.

In cosmos, these two location are all accepted: "West US", "westus". And it will return the "West US" while our Region uses "westus".

Reply:
Ok, then maybe `static` is better.

Reply:
Done